data.sql

```diff
@@ -1,5 +1,10 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,
+neutered,weight_kg) VALUES (1,'Agumon','2020-02-03',0,true,10.23);
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,
+neutered,weight_kg) VALUES (2,'Gabumon','2018-11-15',2,true,8);
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,
+neutered,weight_kg) VALUES (3,'Pikachu','2022-1-7',1,false,15.04);
```

Comment:
- [ ] Kindly make sure the data you enter here is correct, the `date_of_birth` of the animal is 3 is `2021/01/07` not 2022/01/07`

Reply:
Thanks for the insight